third_party/proton/csrc/include/Driver/GPU/Roctracer.h

```diff
@@ -0,0 +1,85 @@
+#ifndef PROTON_DRIVER_GPU_ROCTRACER_H_
+#define PROTON_DRIVER_GPU_ROCTRACER_H_
+
+#include <roctracer/roctracer.h>
+
+namespace proton {
+
+namespace roctracer {
+
+template <bool CheckSuccess>
+roctracer_status_t set_properties(roctracer_domain_t domain, void *properties);
```

Comment:
done

Reply:
This seems like a really bad idea as the calls no longer match the roctracer api.  This was a wrapper to load and use libroctracer.so but now we have renamed every api call in that library.  It will work.  It is confusing.